src/components/POLine/POLineView.js

```diff
@@ -429,10 +436,13 @@ const POLineView = ({
             lineId={line.id}
           />
 
-          <LineLinkedInstances
-            line={line}
-            toggleSection={onToggleSection}
-          />
+          {!line.isPackage && (
```

Comment:
so was it agreed to hide for non-pacakge?

Reply:
We agreed to display Package titles accordion on package POL under Item details accordion and Linked instance accordion for regular POL on the previous position